trust-store-trust-policy-specification.md

```diff
@@ -100,17 +102,19 @@ Since revocation check requires network call and network call can fail because o
 - `fail-open`: If revocation endpoint is not reachable, consider artifact as not revoked.
 - `fail-close`: If revocation endpoint is not reachable, consider artifact as revoked.
 
+### Trust Policy Schema
+
 The trust policy is represented as JSON data structure as shown below:
 
-```json
+```jsonc
 {
     "version": "1.0",
     "trustPolicies": [
         {
             "name": "verify-signature",
             "scopes": [
-                "wabbit-networks.io/software/product1"
-                "wabbit-networks.io/software/product2" ],
+                "registry.wabbit-networks.io/software/net-monitor"
+                "registry.wabbit-networks.io/software/net-logger" ],
             "trustStores": [ "trust-store-name-1", "trust-store-name-2" ],
```

Comment:
Lets update the trust store names to correspond to the example. Need not demonstrate it referencing two trust stores, more confusing for the reader.

Reply:
Why are we adding `registry` as prefix? Is it just part of registry URI?

Reply:
> Why are we adding registry as prefix? Is it just part of registry URI?

Yeah, it's just the registry name. The presumption is wabbit-networks.io is their public website :)

Reply:
> Lets update the trust store names to correspond to the example. Need not demonstrate it referencing two trust stores, more confusing for the reader.

@gokarnim, are you suggesting I sync the trust stores in line 118?